src/libstd/f32.rs

```diff
@@ -835,7 +835,7 @@ impl f32 {
         if self == Self::NEG_INFINITY {
             Self::NEG_INFINITY
         } else {
```

Comment:
This branch can be removed, since the other branch now works for -inf.

Reply:
Good catch!